runtime/intern.c

```diff
@@ -95,8 +95,14 @@ struct caml_intern_state {
   /* 1 if the compressed format is in use, 0 otherwise */
 };
 
-/* Allocates the domain local intern state if needed */
-static struct caml_intern_state* get_intern_state (void)
+static void intern_init_stack(struct caml_intern_state* s)
```

Comment:
`init_intern_stack` is a better name because we're initializing the `intern_stack`. I don't like the `intern` and `stack` being split in the current name.